spec.html

```diff
@@ -122,6 +122,32 @@ <h1>
 		</emu-note>
 	</emu-clause>
 
+	<emu-clause id="sec-create-type-error-copy" type="abstract operation">
+		<h1>
+			CreateTypeErrorCopy (
+				_realmRecord_: a Realm Record,
+				_originalError_: an ECMAScript language value
+			)
+		</h1>
+		<emu-alg>
+			1. Let _newError_ be a newly created *TypeError* object.
+			1. NOTE: _newError_ is created in _realmRecord_.
+			1. If Type(_originalError_) is Object, then
+				1. Let _message_ be Completion(Get(_originalError_, "message")).
+				1. If _message_ is not an abrupt completion, then
+					1. Let _msgString_ be Completion(ToString(_message_)).
```

Comment:
Minor, matter of opinion: My suggestion was, rather than calling ToString, simply check if it's a string and not add the message if it's not. I'm OK with these semantics as well.